### PR TITLE
Implement Thread overview in the SideMenu [full flow]

### PR DESCRIPTION
**Pull request by dissolvent**

### CODE REVIEW

File: frontend/src/scenes/Workspace/containers/Thread/index.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+};
+
+const mapStateToProps = (state: IAppState) => ({
+  // const { workspace: { activeThread: { post, comments } } } = state;
+  post: state.workspace.activeThread?.post,
+  comments: state.workspace.activeThread?.comments
+});
+
+const mapDispatchToProps = {
+  sendComment: addCommentRoutine
```
Comment (AlikRakhmonov) [inline frontend/src/scenes/Workspace/containers/Thread/index.tsx:58]: it should be in src/containers/Thread

File: frontend/src/scenes/Workspace/sagas/index.ts [hunk 2]
```diff
@@ -23,6 +29,50 @@ function* watchPostWorkspaceName() {
   yield takeEvery(addWorkspaceRoutine.TRIGGER, addWorkspaceReq);
 }
 
+function* addCommentRequest({ payload }: Routine<any>) {
```
Comment (AlikRakhmonov) [inline frontend/src/scenes/Workspace/sagas/index.ts:32]: it should be on src/containers/Thread level

File: frontend/src/scenes/Workspace/containers/Thread/index.tsx (hunk 1 above)
Comment (dissolvent) [reply frontend/src/scenes/Workspace/containers/Thread/index.tsx]: Add comment now in src/containers/Thread

File: frontend/src/scenes/Workspace/sagas/index.ts (hunk 2 above)
Comment (dissolvent) [reply frontend/src/scenes/Workspace/sagas/index.ts]: Add comment request now in src/containers/Thread

**Pull request merged**